makefile

```diff
@@ -103,7 +103,8 @@ data/gtfs/%.merged.with_flex.gtfs: data/gtfs/%.merged.gtfs.zip
 	rm -rf $@
 	unzip -d $@ $<
 	$(info patching GTFS-Flex data into the GTFS feed)
-	docker run -i --rm -v $(HOST_MOUNT)/data/gtfs/$(@F):/gtfs derhuerst/generate-herrenberg-gtfs-flex
+	# todo: pick flex rules file based on GTFS feed
```

Comment:
Because AFAIK, currently, there's only one Flex-patched feed (`hbg6`), I have hard-coded the Stadtnavi Herrenberg flex rules file below. But because we will likely have more Flex-patched feeds in the future (e.g. bbnavi's), I've put this note.

Shall we solve this via a column in `config/gtfs-feeds.csv`?

Whatever mechanism we pick, it seems a bit "entangled" with the inner workings of `generate-gtfs-flex`, as it depends on its bundled rule files. We may want to move the bundled files here instead and mount them into `generate-gtfs-flex`.